test/carbon/trading/trading.ts

```diff
@@ -303,6 +305,18 @@ describe('Trading', () => {
         await transfer(deployer, tokens[sourceSymbol], trader, expectedSourceAmount);
     };
 
+    /**
+     * generates a test order
+     */
+    const generateTestOrder = (): TestOrder => {
```

Comment:
this would be the 3rd copy of the same function, move to a shared utility function?

Reply:
exported into a shared utility function